frontend/src/MovieGallery/api/image-api.ts

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-const baseUrl:string = "https://image.tmdb.org/t/p"
+const baseUrl:string = "https://image.tmdb.org/t/p/w500"
```

Comment:
I would save the whole url in the model then you can just pass the url in ur img tag, and dont get in trouble when the base url of some images changes...